README.md

```diff
@@ -1,15 +1,29 @@
 # Engine
 
-## Состав команды
+## Developers
 
 + Мороз Владислав - [Moroz Vladislav](https://github.com/MorozVladislav)
 + Василькова Юля - [Vasilkova Julia](https://github.com/JuliaVasilkova)
 + Борисов Дмитрий - [Borisov Dmitry](https://github.com/stalkerboray)
 
-## Задания
+## Tasks
 
 ### Граф визуальный прекрасный I
-[Задание 1](tasks/task_1.md)
+[Task 1](tasks/task_1.md)
 
 ### Клиент игровой великолепный II
-[Задание 2](tasks/task_2.md)
\ No newline at end of file
+[Task 2](tasks/task_2.md)
+
+## Dependencies
+Application requires python2.7 , Tkinter 8.6 and pip installed. All the rest dependencies are indicated in requirements.txt
```

Comment:
Please delete whitespace before comma and revise Python title
...Python 2.7, Tkinter 8.6....